### ISSUE TITLE
Labels of test data.

### ISSUE THREAD

**Issue by Chuanqi-Zang** [bcmi/Causal-VidQA#1]
I trained the whole model and got the expected result on the **Validation set**. But the result on the **Test set** was 0. I find that the label of the test set is always **-1**. How can I get the test set Label?

**Comment by Jiangtong-Li** [comment]
You can submit your inference results in test set to https://codalab.lisn.upsaclay.fr/competitions/6269 for evaluation. Each account can submit inference results 10 times in total.
